boot/bootutil/src/bootutil_public.c

```diff
@@ -471,25 +471,28 @@ boot_swap_type(void)
 }
 
 /**
- * Marks the image in the secondary slot as pending.  On the next reboot,
- * the system will perform a one-time boot of the the secondary slot image.
+ * Marks the image with the given index in the secondary slot as pending. On the
+ * next reboot, the system will perform a one-time boot of the the secondary
+ * slot image.
+ *
+ * @param image_index       Image pair index.
  *
  * @param permanent         Whether the image should be used permanently or
- *                              only tested once:
- *                                  0=run image once, then confirm or revert.
- *                                  1=run image forever.
+ *                          only tested once:
+ *                               0=run image once, then confirm or revert.
+ *                               1=run image forever.
  *
  * @return                  0 on success; nonzero on failure.
  */
 int
-boot_set_pending(int permanent)
+boot_set_pending_multi_image(int image_index, int permanent)
```

Comment:
The changes look good to me, but how about calling them `boot_set_pending_multi` and `boot_set_confirmed_multi`  (no image postfix) so they both match the naming scheme used for `boot_swap_type_multi`?

Reply:
Done